flatbuffers/ExecutionReport.fbs

```diff
@@ -55,6 +55,8 @@ table ExecutionReport {
     time_in_force: TimeInForce;
     /// See ExecType.
     exec_type: ExecType;
+    /// See ExecInst.
+    exec_inst: ExecInst;
```

Comment:
You can only add to the bottom of the list in flatbuffers, move to the end of the table